ansible/roles/common/files/path.sh

```diff
@@ -0,0 +1 @@
+export PATH=$PATH:/usr/local/flink/bin
```

Comment:
I think it is a bad practice to hard code the installation path (usr/local) of Apache Flink. Have you considered using the environment template file the common role already deployes?

https://github.com/grnet/okeanos-LoD/blob/master/ansible/roles/common/templates/environment.j2